tools/pipelines/test-stability.yml

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# test-stability pipeline (for the purpose of detecting intermittent and flaky failures)
+
+name: $(Build.BuildId)
+
+trigger: none
+pr: none
+
+parameters:
+- name: poolBuild
+  type: object
+  default: Large
+
+- name: stageNames
```

Comment:
would this be better as a variable, rather than a parameter?

Reply:
The `poolBuild` is used as a parameter passed to the template `include-test-stability.yml`. I suppose the template can not accept input variable in terms of syntax. 

And as a reference I found the poolBuild is used as a parameter in other test pipelines as well
https://github.com/microsoft/FluidFramework/blob/0c0f76f61a27b481e0eec7378b2bae6b4c227773/tools/pipelines/test-real-service.yml#L49

Reply:
I think Tony might have meant `stageNames`. Making that one a variable means we wouldn't be able to change (when kicking off the pipeline) how many iterations are done, but that seems fine for the purposes of this pipeline?

Reply:
Ahh I see... Actually I thought we might change the times of iterations in the future, so I just set it as a parameter. Any ideas Tony? @anthony-murphy

Reply:
i prefer not to add complexity when we don't know if we'll need it. i'd rather not expose it at this point, so i think a variable i a better fit

Reply:
According to [this doc](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch) the variable cannot store an array, it will pass the content as a string to the appsettings.json file. What I can think of is storing all stage names as a string in the variable and use split/trim when the single element inside is needed, but this method may lack readability? Considering we need to define a very long string `stageNames="Test1,Test2,....,Test100"` and split by ','